CHANGELOG.md

```diff
@@ -12,11 +12,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Frost Trees will now drop saplings
 - Saplings are now pottable
 - Aspect slabs can now be used in crafting recipes
+- Uranium button now made with 1 raw uranium instead of 1 uranium block
+- Removed default name visibility of Lotus Flower
```

Comment:
I recall seeing this issue in 1.20.4, but was this also an issue back in 1.20.1?

Reply:
Tested it out just now, and the 1.20.4 issue does not appear to be present in 1.20.1 with Minestuck only. Unless there's a specific reason to this for 1.20.1, I think that we should revert this and the `LotusFlowerEntity` change before merging this.